tests/test_opt.py

```diff
@@ -296,7 +296,7 @@ def test_merge_common(self):
         graph = helper.make_graph(new_nodes, 'test0', [input0], [output0])
         model = helper.make_model(graph)
         onnx.save_model(model, self.get_temp_file('temp_after.onnx'))
-        self.assertEqual(len(new_nodes), 8)
+        self.assertEqual(len(new_nodes), 11)
```

Comment:
Does this test cover the new Apply function you added?

Reply:
some tests are covered in keras2onnx.